lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -636,3 +636,39 @@ function dosomething_campaign_get_runs($nid, $language) {
 
   return $campaign_runs;
 }
+
+/**
+ * Get all the affiliate configurations data for a campaign.
+ *
+ * @param  string|array $campaign_info Either a node ID or an array of campaign variables.
+ * @return array
+ */
+function dosomething_campaign_get_affiliate_data($campaign_info) {
+  if (!$campaign_info) {
+    return FALSE;
```

Comment:
Let's return `NULL` rather than `FALSE` on failure, and update the docblock to `@return array|null`.